Neos.Flow/Tests/Behavior/Features/Bootstrap/SecurityOperationsTrait.php

```diff
@@ -95,11 +95,6 @@ public function iHaveTheFollowingPolicies($string)
         self::$testingPolicyPathAndFilename = $this->environment->getPathToTemporaryDirectory() . 'Policy.yaml';
         file_put_contents(self::$testingPolicyPathAndFilename, $string->getRaw());
 
-        $configurationManager = $this->objectManager->get(ConfigurationManager::class);
-        $configurations = ObjectAccess::getProperty($configurationManager, 'configurations', true);
-        unset($configurations[ConfigurationManager::CONFIGURATION_PROCESSING_TYPE_POLICY]);
-        ObjectAccess::setProperty($configurationManager, 'configurations', $configurations, true);
```

Comment:
I don't understand why this was done until now, but this is really not needed any more?

Reply:
In short, this code does nothing and is basically a typo which exists very long in the current master. In the current master (and in this PR) the key of `$configurations` is the configurationType, not the configurationProcessingType. Thus the `unset(...)` does nothing. For example, see https://github.com/neos/flow-development-collection/blob/7b800d8217b30a5e6b96f0c648aec034f82bc5ff/Neos.Flow/Classes/Configuration/ConfigurationManager.php#L379.